examples/multiple-apps.ts

```diff
@@ -46,8 +46,13 @@ import { pause, startAnalysis } from '../src/index';
 
         await appAnalysis.stopTrafficCollection();
 
-        const res = await appAnalysis.stop();
-        console.log(res);
+        const result = await appAnalysis.stop();
+
+        console.log(result);
+        // {
+        //    app: { id: '<app id>', version: '<app version>' },
+        //    traffic: { '2023-03-27T10:29:44.197Z': { log: [Object] } }
```

Comment:
You are still not mentioning the interesting part: That the traffic is a HAR. :D